wallet/rescan.go

```diff
@@ -63,14 +63,14 @@ func NewRescanFilter(addresses []dcrutil.Address, unspentOutPoints []*wire.OutPo
 }
 
 // AddAddress adds an address to the filter if it does not already exist.
-func (f *RescanFilter) AddAddress(a dcrutil.Address) {
+func (f *RescanFilter) AddAddress(a stdaddr.Address) {
 	switch a := a.(type) {
-	case *dcrutil.AddressPubKeyHash:
+	case *stdaddr.AddressPubKeyHashEcdsaSecp256k1V0:
 		f.pubKeyHashes[*a.Hash160()] = struct{}{}
-	case *dcrutil.AddressScriptHash:
+	case *stdaddr.AddressScriptHashV0:
 		f.scriptHashes[*a.Hash160()] = struct{}{}
-	case *dcrutil.AddressSecpPubKey:
-		serializedPubKey := a.ScriptAddress()
+	case *stdaddr.AddressPubKeyEcdsaSecp256k1V0:
+		serializedPubKey := a.SerializedPubKey()
```

Comment:
Might as well remove the `case 65` below because uncompressed pubkeys are never actually used and I'd like to make them nonstandard even.

Reply:
yes, i had thought about doing that too. wasn't sure if i wanted to do it as part of this commit in order to keep the review easier but will since you brought it up.